poolrpc/trader.proto

```diff
@@ -870,6 +870,29 @@ enum MatchRejectReason {
 }
 
 message RecoverAccountsRequest {
+    /*
+    Recover the latest account states without interacting with the
+    Lightning Labs server.
+    */
+    bool full_client = 1;
+
+    /*
+    Number of accounts that we are trying to recover. Used during the
+    full_client recovery process.
+    */
+    uint32 account_target = 2;
+
+    /*
+    Auctioneer's public key. Used during the full_client recovery process.
```

Comment:
I will leave it without the comment for the height hint because it could be used in the future to narrow the searches if you know approx when you created your account.